src/main/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWater.java

```diff
@@ -28,7 +28,7 @@
 	//lev_acy_cd
 	protected String     levelAccuracyCode;
 	//lev_status_cd
-	protected String     siteStatusCode;
+	protected String     readingQualifiers;
```

Comment:
This field now initially holds the multi-valued JSON string from Aquarius, then when the rules are applied, it is converted to a single digit code that will be used by NWISWeb.  The new digit codes are a new thing - they were single letters before.